src/main/kotlin/no/nav/familie/ef/mottak/service/ITextSøknadService.kt

```diff
@@ -53,6 +54,7 @@ class ITextSøknadService(
         val lagretSkjema = søknadRepository.insert(søknadDb)
         vedleggRepository.insertAll(vedlegg)
         iTextPdfService.lagITextPdf(lagretSkjema.id)
+        taskProsesseringService.startPdfKvitteringTaskProsessering(søknadDb)
```

Comment:
Du er nødt til å fjerne `iTextPdfService.lagITextPdf(lagretSkjema.id)` fra linjen rett over her så dette ikke gjøres to ganger